src/Cli/Microsoft.DotNet.Cli.Utils/ExponentialRetry.cs

```diff
@@ -43,16 +41,40 @@ public static IEnumerable<TimeSpan> Intervals
                     return result;
                 }
 
-                log?.WriteLine($"Operation failed. Retry count: {count}");
                 count++;
                 if (count == maxRetryCount)
                 {
-                    throw new RetryFailedException(
+                    throw new Exception(
                         $"Retry failed for {taskDescription} after {count} times with result: {result}");
```

Comment:
This will need to be localized now.  Previously it was just used to retry tests, so it didn't need to be localized, but now we are using it in the product.

Reply:
Should we perhaps include a message informing the user of the interval? The first might be fine, but jumping exponentially without any feedback might make them think that the command is unresponsive

Reply:
Good catch. Although we have no time for loc pass. We can format is differently and considering this as a dev message. At the end of the message

Reply:
ideally the user should not see it all the time

Reply:
I've refactored so we shouldn't need the error anymore